src/NServiceBus.Core.Tests.x86/When_running_under_a_32Bit_process.cs

```diff
@@ -1,22 +0,0 @@
-﻿namespace NServiceBus.Core.Tests.x32
-{
-    using System.Linq;
-    using Hosting.Helpers;
-    using NUnit.Framework;
-
-    [TestFixture]
-    public class When_running_under_a_32Bit_process
-    {
-        [Test]
-        public void Should_skip_x64_assemblies()
```

Comment:
@andreasohlund This is testing a specific code path in the assembly scanner: https://github.com/Particular/NServiceBus/blob/0b64b3507678b4473d4fb26d59bf465e8d0121ed/src/NServiceBus.Core/Hosting/Helpers/AssemblyScanner.cs#L147

Might be worth keeping the test for that?